### PR TITLE
Allows vampire thralls to be de-thralled using holy water or bibles.

### LINKED ISSUE #7804: There is no way to unthrall people
>Thrall sec
>Even if the chaplain knows about it, there is nothing he can do, even if the vampire dies.

### PR DESCRIPTION
**Pull request by gbasood**
Flavor text copypasted please give better suggestions.
Closes #7804

### CODE REVIEW

**Comment by Ephx1** [comment]
Adding onto this I don't think loyalty implants prevent thralling, if you believe that to be an issue as well.

**Comment by ghost** [comment]
When the vampire is ashed, should those thralled to him be freed?

**Comment by gbasood** [comment]
Pretty sure they already are. Aren't they?

**Comment by Probe1** [comment]
:+1:

**Comment by gbasood** [comment]
Okay they don't. Not that I can find, anyway.
I also can't find where vamps actually get ashed, or if they get special handling for getting ashed, specifically.

**Comment by Imagebard** [comment]
:+1:

**Comment by AutismSpider** [comment]
@Ephx1 Thralling goes right past loyalty implants

**Comment by Ephx1** [comment]
Yeah, that's why I mentioned it. It's resistant to everything including the powers of a Dark God, but not some undead chump's stare.

**Comment by Imagebard** [comment]
>It's resistant to everything

Actually it pretty much sucks dick and only has two functions in the game. Loyalty Implant worst implants.

**Comment by Ephx1** [comment]
I meant in the terms of changing your allegiance.
Which is true, there's only three allegiance changing procedures, but I still think it should cover all three.

File: code/game/objects/items/weapons/storage/bible.dm [hunk 1]
```diff
@@ -124,6 +124,10 @@
 				ticker.mode.remove_cultist(H.mind)
 			else //We aren't deconverting him this time, give the Cultist a fair warning
 				to_chat(H, "<span class='warning'>The power of [deity_name] is overwhelming you. Your mind feverishly questions Nar'Sie's teachings!</span>")
+		if(H.mind && H.mind.special_role == "VampThrall")
+			ticker.mode.remove_vampire_mind(H.mind, H.mind)
+			H.visible_message("<span class='notice'>[H] suddenly becomes calm and collected again, his eyes clear up.</span>",
```
Comment (Intigracy) [inline code/game/objects/items/weapons/storage/bible.dm]: /his eyes, same on other one

or is it \his I can't remember w/e

**Comment by CptWad** [comment]
Thank you :pray:  j e s u s :pray: 
pun intended

File: code/game/objects/items/weapons/storage/bible.dm (hunk 1 above)
Comment (Exxion) [inline code/game/objects/items/weapons/storage/bible.dm]: `\his`
All macros use backslashes

**Comment by Probe1** [comment]
Loyalty implants are very useful for keeping track of authorized people, like a cult icon for cultists.

Anyhow vampires its own thing and hey whats wrong with not having the same rules for every game mode.  Its fine.

**Comment by Ephx1** [comment]
Because there is a point where Vamp becomes its own rev/cult, except with one converter. Where there's just a mob of people dragging them to the vamp for conversion.

Just something I thought people might want changed.

**Comment by clusterfack** [comment]
Someone really should make loyalty implants prevent being enthralled

**Pull request merged**